autonomi/src/client/archive.rs

```diff
@@ -43,6 +76,58 @@ impl Archive {
 
         Ok(root_serialized)
     }
+
+    /// Rename a file in an archive
+    /// Note that this does not upload the archive to the network
+    pub fn rename_file(&mut self, old_path: &Path, new_path: &Path) -> Result<(), ArchiveError> {
+        let data_addr = self
```

Comment:
This would not be just the `data_addr`, maybe use generic `key`?

Reply:
Indeed, renamed to value as more accurate!